addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1133,7 +1133,7 @@ const UnitUserValueWidget = UserValueWidget.extend({
         const activeValue = this._super(...arguments);
 
         const params = this._methodsParams;
-        if (!params.unit) {
+        if (!params.unit && !params.fakeUnit) {
```

Comment:
Seems a bit fragile:
- Why would only this function have to be changed: I see the other ones (`getDefaultValue`, `setValue`, ...) also perform manipulation of float vs string. E.g. `setValue` does `this._floatToStr(numValue)` -> which should probably also be done if a fakeUnit is used? I'd say we should rather create a `hasUnit` param/function or something like that?
- `fakeUnit` seems to be a feature of `InputUserValueWidget`, not its parent class `UnitUserValueWidget`, probably another good reason to create a `hasUnit` function to be overridden over there?

Reply:
For tracking purpose: someone will take over this PR